core/src/main/java/jenkins/security/ApiTokenProperty.java

```diff
@@ -355,6 +356,34 @@ public ApiTokenStats getTokenStats() {
         return tokenStats;
     }
 
+    // essentially meant for scripting
+    public @Nonnull String addFixedNewToken(@Nonnull String name, @Nonnull String tokenPlainValue) throws IOException {
```

Comment:
All of these should be `@Restricted` unless they're meant to be available to plugins.